### ISSUE TITLE
ANDI violation for Progress and StackedProgress component. 

### ISSUE THREAD

**Issue by Jithinqw** [Jithinqw/wafflecss#32]

**Comment by Jithinqw** [comment]
#48 targeted ANDI accessibility scripts.

**Comment by Jithinqw** [comment]
Fixed in latest release v1.0.47
